app/javascript/src/NewService/components/FormElements/ErrorMessage.jsx

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import React from 'react'
+
+type Error = {
+  hasError: boolean,
+  errorMsg: string
+}
+
+const ErrorMessage = ({error}: {error: Error}) => error.hasError
```

Comment:
Maybe it's better to have as the sole prop the `errorMsg` string and check if it has error before rendering the component. Thus this component will always return a jsx, as it should be.